cmd/execution/main.go

```diff
@@ -564,6 +570,81 @@ func main() {
 		}).Run()
 }
 
+func logEpochNumber(vm *fvm.VirtualMachine, vmCtx fvm.Context, executionState state.ExecutionState, node *cmd.NodeConfig, log zerolog.Logger) {
+	logFail := func(err error) {
+		log.Error().Err(err).Msg("could not log epoch number")
+	}
+
+	protocolStateEpoch, err := node.State.
+		AtBlockID(node.RootBlock.ID()).
+		Epochs().
+		Current().
+		Counter()
```

Comment:
I am not 100% on this part. Can someone confirm that this is the correct way to get the protocol state epoch counter.